tests/c/simple_interp_loop2.newcg.c

```diff
@@ -0,0 +1,113 @@
+// ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// Run-time:
+//   env-var: YKD_LOG_JITSTATE=-
+//   env-var: YKD_LOG_STATS=/dev/null
+//   stderr:
+//     jitstate: start-tracing
+//     pc=0, mem=4
+//     pc=1, mem=4
+//     pc=2, mem=4
+//     pc=3, mem=3
+//     jitstate: stop-tracing
+//     pc=0, mem=3
+//     pc=1, mem=3
+//     pc=2, mem=3
+//     pc=3, mem=2
+//     jitstate: enter-jit-code
+//     pc=0, mem=2
+//     pc=1, mem=2
+//     pc=2, mem=2
+//     pc=3, mem=1
+//     pc=0, mem=1
+//     pc=1, mem=1
+//     pc=2, mem=1
+//     pc=3, mem=0
+//     jitstate: deoptimise
+//     pc=4, mem=0
+//     pc=5, mem=0
+
```

Comment:
Does this need a description what this test does (since we have one for all other tests)?

Reply:
The lack of description was inherited from the old codegen test. I added commits to fix both.